wellcomeml/viz/visualize_clusters.py

```diff
@@ -0,0 +1,31 @@
+from bokeh.plotting import figure, show
+from wellcomeml.viz.palettes import Wellcome33
+from bokeh.io import output_notebook
+output_notebook()
+
+
+def visualize_clusters(reduced_points: list):
+
+    """
+    Visualises clusters and shows basic information
+    Args:
+        reduced_points: list
+        List of list of reduced points. Available at cluster.reduced_points
+
+    Returns:
+        p
```

Comment:
It actually returns None. You can just say: Returns: None (prints a bokeh figure to file or to notebook)